apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -869,23 +869,23 @@ When set to true, invalid utf8 strings in for example client ID, topic name, etc
 
     mqtt_max_awaiting_rel {
         desc {
-            en: """Maximum QoS 2 packets (Client -> Broker) awaiting PUBREL."""
-            zh: """PUBREL (Client -> Broker) 最大等待队列长度。"""
+            en: """For each publisher session, the maximum number of outstanding QoS 2 messages pending on the client to send PUBREL. After reaching this limit, new QoS 2 PUBLISH requests will be rejected with `147(0x93)` until either PUBREL is received or timed out."""
+            zh: """每个发布者的会话中，都存在一个队列来处理客户端发送的 QoS 2 消息。该队列会存储 QoS 2 消息的报文 ID 直到收到客户端的 PUBREL 或超时，达到队列长度的限制后，新的 QoS 2 消息发布会被拒绝，并返回 `147(0x93)` 错误。"""
         }
         label: {
             en: """Max Awaiting PUBREL"""
-            zh: """Max Awaiting PUBREL"""
+            zh: """PUBREL 等待队列长度"""
         }
     }
 
     mqtt_await_rel_timeout {
         desc {
-            en: """The QoS 2 messages (Client -> Broker) will be dropped if awaiting PUBREL timeout."""
-            zh: """PUBREL (Client -> Broker) 最大等待时间，超时则会被丢弃。"""
+            en: """The QoS 2 messages (Client -> Broker) handle flow will be ignored if awaiting PUBREL timeout. This QoS 2 message can be published successfully."""
```

Comment:
```suggestion
            en: """For client to broker QoS 2 message, the time limit for the broker to before the last PUBREL message is received, the wait will be aborted after timed out. Note, the message is delivered to subscribers before entering the wait for PUBREL."""
```

Reply:
Yes, it is https://github.com/emqx/emqx/blob/c01eae5a8ba330b3b4c7f5e4876eab70594e7dae/apps/emqx/src/emqx_channel.erl#L476-L484